i18n/pt-br/README.md

```diff
@@ -0,0 +1,15 @@
+# Código de Conduta para Colaboradores do Statik
+
+Veja em [CODE_OF_CONDUCT.md](https://github.com/statikstack/code-of-conduct/blob/master/CODE_OF_CONDUCT_PTBR.md).
```

Comment:
@AlyoshaS O endereço do arquivo `CODE_OF_CONDUCT.md` está incorreto. O correto seria o endereço do arquivo que está no diretório da tradução: `https://github.com/statikstack/code-of-conduct/blob/master/i18n/pt-br/CODE_OF_CONDUCT.md`.

Reply:
Feito.